llama_index/evaluation/scoring/semantic_answer_similarity.py

```diff
@@ -0,0 +1,46 @@
+class SemanticAnswerSimilarity:
+    def __init__(
+        self,
+        model: str = "cross-encoder/stsb-roberta-large",
+    ) -> None:
+        try:
+            from sentence_transformers import CrossEncoder
+        except ImportError:
+            raise ImportError(
+                "Cannot import sentence-transformers package,",
+                "please `pip install sentence-transformers`",
+            )
+
+        self._model = CrossEncoder(model, max_length=512)
+
+    def evaluate(self, ground_truth: str, predicted: str) -> float:
+        """
+        Evaluates the semantic similarity between a ground truth answer and
+        a predicted answer.
+        """
+        scores = self._model.predict([(ground_truth, predicted)])
+        return scores[0]
+
+
+class SemanticRelationMatch:
+    def __init__(
+        self,
+        model: str = "cross-encoder/nli-deberta-v3-base",
+    ) -> None:
+        try:
+            from sentence_transformers import CrossEncoder
+        except ImportError:
+            raise ImportError(
+                "Cannot import sentence-transformers package,",
+                "please `pip install sentence-transformers`",
+            )
+
+        self._model = CrossEncoder(model, max_length=512)
+
+    def evaluate(self, ground_truth: str, predicted: str) -> float:
+        """
+        Evaluates the semantic similarity between a ground truth answer and
+        a predicted answer.
+        """
```

Comment:
This has the same doctoring as `SemanticAnswerSimilarity`  -- what's the actual difference here? Why would I use one vs. the other?

I see that this is an NLI model (i.e. predicts entails, contradicts, neutral), but might be good to better convey to a user how this differs from normal similarity 

Some possible ideas:
- `SemanticAnswerAgreement`
- I thought I had more idea but tbh I'm drawing blanks haha